core/log/logger.go

```diff
@@ -0,0 +1,9 @@
+package log
+
+type Logger interface {
+	Debug(msg string, keyvals ...interface{})
+	Info(msg string, keyvals ...interface{})
+	Error(msg string, keyvals ...interface{})
+
+	With(keyvals ...interface{}) Logger
```

Comment:
Should With be exposed here?

As I see it, With is only used to namespace module, e.g. `With("module", "x/group")`. Maybe instead runtime/depinject can set .With() on tendermint's logger when creating this service for each module. This way app developers don't need to care about With anymore.

Reply:
we can do this, but since we pass the logger to tendermint then this would be their expected keeper. We can have our implementation adhere to both as an alternative?

Reply:
Yeah okay, since people are used to TM's logger, maybe it's better to not add a layer of runtime/depinject doing its own things with the logger.